settings/src/commonMain/kotlin/com/islamversity/settings/licenses/LicensesState.kt

```diff
@@ -0,0 +1,14 @@
+package com.islamversity.settings.licenses
+
+import com.islamversity.core.mvi.BaseState
+import com.islamversity.core.mvi.BaseViewState
+import com.islamversity.settings.models.LicensesUIModel
+
+data class LicensesState(
+    override val base: BaseState,
+    val licenses: List<LicensesUIModel>
```

Comment:
preferably `private`

Reply:
Can you please explain to me more?
I need to access it from LicesnsesView and nowhere in the code a State Class property is set to private

Reply:
Yeah my bad sorry
you are 100% correct